packages/protocol/scripts/sourcify-publish.ts

```diff
@@ -33,42 +33,26 @@ import Web3 from 'web3'
  *
  */
 
-interface BuildOptions {
-  network: string
-  buildArtifactsPath: string
-  proposalPath: string
+const _buildTargets = {
+  network: undefined,
+  build_artifacts_path: undefined,
+  proposal_path: undefined,
 }
 
-async function main(buildTargets: BuildOptions) {
-  const artifactBasePath = buildTargets.buildArtifactsPath || './build/contracts'
+async function main(buildTargets: typeof _buildTargets) {
+  const artifactBasePath = buildTargets.build_artifacts_path || './build/contracts'
   const artifactPaths = fs.readdirSync(artifactBasePath)
-  const reportPath = buildTargets.proposalPath || './proposal.json'
+  const reportPath = buildTargets.proposal_path || './proposal.json'
   const report = require(path.join(process.cwd(), reportPath))
```

Comment:
Not related to this PR, but can we change report -> proposal to be more accurate with the naming?